lib/components/Select.tsx

```diff
@@ -23,6 +23,18 @@ export default function Select({
   const { fieldName, registerField, defaultValue, error } = useField(name);
   const [value, setValue] = useState<string | string[]>();
```

Comment:
Hey @CesarDenis, first of all, thanks for your contribution 😄 

I was looking at this issue yesterday. And I think simply setting de default value when calling `useState` would solve the issue, like this:

```js
const [value, setValue] = useState<string | string[]>(defaultValue);
```

I think that there is no need to check if the `<select>` is `multiple` or not, given the fact that if the user decides to use `multiple` he has to pass an array of elements as `defaultValue`, Unform should not be responsible for checking that.

What are your thoughs on that?

Reply:
Hey @jpdemagalhaes, I agree to remove the responsibility of check is `multiple` or not.

But, we do not need a check? Because when the value is not in the initial data, the value is `undefined`. Could not this cause some error in form validation?

Reply:
Nice catch! I agree that it should not return `undefined`.

Either an empty string or an empty array when `initialData` is not passed.

Maybe something like this could work:
```js
  const [value, setValue] = useState<string | string[]>(
    defaultValue || (multiple ? [] : "")
  );
```

Feel free to propose changes if you have a different vision than me.